src/frontend/src/routes/(new-styling)/unsupported/+page.svelte

```diff
@@ -1,8 +1,45 @@
 <script lang="ts">
   import XBrowser from "./components/views/XBrowser.svelte";
   import Generic from "./components/views/Generic.svelte";
+  import { afterNavigate, beforeNavigate, goto } from "$app/navigation";
+  import type { AfterNavigate, BeforeNavigate } from "@sveltejs/kit";
+
+  const SESSION_KEY = "ii-unsupported-page-reloaded";
```

Comment:
Nit. The variable name implies that this is a session-specific key, whereas we have a constant key for this page. Consider renaming this, e.g.

```suggestion
  const PAGE_RELOAD_KEY = "ii-unsupported-page-reloaded";
```